nightlies_watcher/hg_mozilla.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+
+def get_minimal_repository_name(repository):
+    return repository.split('/')[-1]
+
+
+def get_push_id(repository, revision):
+    response = requests.get(_get_push_log_url(repository, revision))
```

Comment:
10s of timeout added (hardcoded). I'm considering to make these call async, I'll add the redo mechanism at that time.